app/controllers/api/v0/comments_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Api
+  module V0
+    class CommentsController < Api::V0::BaseController
+      before_action only: :index do
+        require_access_token %w(read)
+      end
+
+      before_action only: %i(create destroy) do
+        require_access_token %w(read write)
+      end
+
+      rescue_from ActiveRecord::RecordNotFound do
+        render json: I18n.t("comments.not_found"), status: 404
```

Comment:
The comments/posts routes will be removed from this PR before merge – they still contain the issues you mentioned a while ago.